build.gradle

```diff
@@ -1,90 +1,72 @@
 buildscript {
-  ext.okhttpclientVersion="3.11.0"
```

Comment:
not related to cap-21, but upgraded to latest gradle build, was able to user leaner config for shadowing jar.